_data/software/beagle-genotype-imputation-method.yml

```diff
@@ -0,0 +1,36 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      Beagle is a software package for phasing genotypes and for imputing
+      ungenotyped markers.
+    fr: >-
+      Beagle est un progiciel pour phaser les génotypes et imputer des marqueurs
+      non génotypés.
+category: '43232600'
+homepageURL:
+  en: 'https://faculty.washington.edu/browning/beagle/beagle.html'
+  fr: 'https://faculty.washington.edu/browning/beagle/beagle.html'
+licences:
+  - URL:
+      en: 'https://faculty.washington.edu/browning/beagle/beagle.html'
```

Comment:
Looks like they have this link on that page too https://faculty.washington.edu/browning/beagle/gpl_license, but pointing here works too since it describes the "or later" part